test/support/utils_sycl.h

```diff
@@ -259,7 +271,7 @@ void
 test3buffers(int mult = 1)
 {
     const sycl::queue& queue = my_queue; // usm requires queue
-#if _PSTL_SYCL_TEST_USM
+    if(has_aspect(queue.get_device(), sycl::aspect::usm_shared_allocations))
```

Comment:
Is it correct that we continue this function in case when this condition is false ?

Reply:
Let's continue conversation on the same issues here: https://github.com/oneapi-src/oneDPL/pull/420#discussion_r769579815